### DISCUSSION TITLE
TL-WN821N V5 not working on big sur.

### DISCUSSION THREAD

**Discussion by miniwing** [chris1111/Wireless-USB-OC-Big-Sur-Adapter#85] [General]
TL-WN821N V5 not working on big sur.

**Comment by chris1111** [comment]
> TL-WN821N V5 not working on big sur.

 TL-WN821N V5 working Operating System: Mac OS 10.6-10.12

See on page
![Screen Shot 2021-07-20 at 11 35 32 PM](https://user-images.githubusercontent.com/6248794/126426770-5ccd1f7e-91bd-476b-8d66-c54844631034.png)
